### ISSUE TITLE
error-message: false positive if Error shadowed

### ISSUE THREAD

**Issue by unlight** [sindresorhus/eslint-plugin-unicorn#1431]
`error-message` false positive if Error shadowed

```ts
import Error from 'rerror';

// wrong: Error message should be a string
const err = new Error({
    name: 'Unauthorized',
});
```

**Comment by fisker** [comment]
Add this https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2d052526d2f8e926487a957bb27e8a88e95b96a5/rules/new-for-builtins.js#L18-L20 before https://github.com/sindresorhus/eslint-plugin-unicorn/blob/2d052526d2f8e926487a957bb27e8a88e95b96a5/rules/error-message.js#L30 will fix this, would you like to work on it?
